packages/psss/src/store/auth.js

```diff
@@ -68,21 +68,12 @@ export const checkIsSuccess = ({ auth }) => auth.status === 'success';
 export const checkIsError = ({ auth }) => auth.status === 'error';
 export const checkIsLoggedIn = state => !!getCurrentUser(state) && checkIsSuccess(state);
 
-export const getPermittedEntity = state => {
-  const user = getCurrentUser(state);
-  return getPermittedEntitiesForUser(user);
-};
-
-export const checkIsMultiCountryUser = state => {
-  const user = getCurrentUser(state);
-  return checkIsAuthorisedForMultiCountry(user);
-};
+export const getEntitiesAllowed = ({ auth }) => getEntitiesAllowedByUser(auth.user);
```

Comment:
Would be nice to put a cache around this - it's used in a lot of places. Perhaps using `reselect`?

Reply:
Yes good suggestion. I have added reselect. I also realised that I could simplify the auth selectors slightly to not need the auth utils and delete that file which I think makes the code simpler.